src/components/views/elements/PersistentApp.tsx

```diff
@@ -71,6 +83,17 @@ export default class PersistentApp extends React.Component<IProps, IState> {
         });
     };
 
+    private onWidgetAction = (payload: ActionPayload): void => {
```

Comment:
This seems to be called whenever any action is dispatched even though the switch statement only cares about widget actions, so I think regular `onAction` would be a better name

Reply:
This is fixed in the [follow up PR](https://github.com/matrix-org/matrix-react-sdk/pull/7450). ( to save me the work of debasing with merge conflicts, I hope that is fine.)